### PR TITLE
Remove 'official' mentions

### LINKED ISSUE #83: Replace 'official' words to 'unofficial'

### PR DESCRIPTION
**Pull request by berkus**
Also update license header in all files.

Closes #83 and ends the milestone

### CODE REVIEW

File: Telegram/Resources/langs/lang.strings [hunk 1]
```diff
@@ -163,7 +163,7 @@ Copyright (c) 2014-2017 John Preston, https://desktop.telegram.org
 "lng_pinned_unpin" = "Unpin";
 "lng_pinned_notify" = "Notify all members";
 
-"lng_intro_about" = "Welcome to the official Telegram Desktop app.\nIt's fast and secure.";
+"lng_intro_about" = "Welcome to the unofficial Telegram Desktop app.\nIt's fast and secure.";
```
Comment (crackedmind) [inline Telegram/Resources/langs/lang.strings]: Maybe  ?
```suggestion
"lng_intro_about" = "Welcome to the "Welcome to the Kepka - unofficial telegram app.\nIt's fast and secure.";
```

Comment (berkus) [reply Telegram/Resources/langs/lang.strings]: ok, sec

Comment (berkus) [reply Telegram/Resources/langs/lang.strings]: ✅

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.192-jfypidax completed](https://ci.appveyor.com/project/procxx/tdesktop/builds/19873439) (commit https://github.com/procxx/kepka/commit/22ea9a5245 by @berkus)

**Comment by crackedmind** [comment]
:x: [Build kepka 1.0.193-qrfwkiiv failed](https://ci.appveyor.com/project/procxx/tdesktop/builds/19875531) (commit https://github.com/procxx/kepka/commit/103ccb812c by @berkus)

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.193-rmrprqtp completed](https://ci.appveyor.com/project/procxx/tdesktop/builds/19876464) (commit https://github.com/procxx/kepka/commit/5a6587a09d by @berkus)

**Review by ForNeVeR — DISMISSED** [review]
License got reworded in some places (because the "special exception" wasn't stated everywhere), but that's okay.

**Comment by crackedmind** [comment]
Need to update kepka.appdata.xml

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.194-wkcyfrvv completed](https://ci.appveyor.com/project/procxx/tdesktop/builds/19943732) (commit https://github.com/procxx/kepka/commit/1eb440db6f by @berkus)

**Review by Randl — DISMISSED** [review]
LGTM

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.195-nabwissm completed](https://ci.appveyor.com/project/procxx/tdesktop/builds/19954799) (commit https://github.com/procxx/kepka/commit/5c29a8a352 by @berkus)

**Review by crackedmind — APPROVED** [review]

**Pull request merged**